File: KITTY/KURS_1/JAVA/DAY_29_3_FRAGMRNT/app/src/main/java/com/example/velic/day_29_3_fragmrnt/Fragment2.java
package com.example.velic.day_29_3_fragmrnt;

import android.app.Fragment;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.TextView;




public class Fragment2 extends Fragment {
    private TextView mInfoTextView;
    private ImageView mCatImageView;
    private String[] mCatDescriptionArray;

    @Override
    public View onCreateView( LayoutInflater inflater,  ViewGroup container,  Bundle savedInstanceState) {
        // return super.onCreateView(inflater, container, savedInstanceState);
        View rootView =
                inflater.inflate(R.layout.fragment2, container, false);
        mInfoTextView = (TextView) rootView.findViewById(R.id.textView);
        mCatImageView = (ImageView) rootView.findViewById(R.id.imageView);

        // загружаем массив из ресурсов
        mCatDescriptionArray = getResources().getStringArray(R.array.cats);
        return rootView;
    }
    public void setDescription(int buttonIndex) {
        String catDescription = mCatDescriptionArray[buttonIndex];
        mInfoTextView.setText(catDescription);

        switch (buttonIndex) {
            case 1:
                mCatImageView.setImageResource(R.drawable.chicken);
                break;
            case 2:
                mCatImageView.setImageResource(R.drawable.dog);
                break;
            case 3:
                mCatImageView.setImageResource(R.drawable.duck);
                break;

            default:
                break;
        }
    }
}

File: KITTY/KURS_1/KOTLIN/DAY_29_2_FRAGMENT_kt/app/src/main/java/com/example/day_29_2_fragment_k/Fragment2.kt
package com.example.day_29_2_fragment_k

import androidx.fragment.app.Fragment
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import android.widget.TextView
import kotlinx.android.synthetic.main.fragment2.*
import com.example.day_29_2_fragment_k.R.drawable.*
import com.example.day_29_2_fragment_k.R.string.*




class Fragment2 : Fragment() {
    private var mInfoTextView: TextView? = null
    private var mCatImageView: ImageView? = null
    private var mCatDescriptionArray: Array<String>? = null
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        // return super.onCreateView(inflater, container, savedInstanceState);
        val rootView = inflater.inflate(R.layout.fragment2, container, false)
        mInfoTextView = rootView.findViewById<View>(R.id.textView) as TextView
        mCatImageView = rootView.findViewById<View>(R.id.imageView) as ImageView

        // загружаем массив из ресурсов
        mCatDescriptionArray = resources.getStringArray(R.array.cats)
        return rootView
    }

    //Подготовим метод (вызывается из Main),
    // который будет менять содержимое фрагмента
    // в зависимости от индекса нажатой кнопки:
    fun setDescription(buttonIndex: Int) {
        // в стоку подключается текст из массива из ресурсов с номером buttonIndex
        val catDescription = mCatDescriptionArray!![buttonIndex]
        mInfoTextView!!.text = catDescription

        when (buttonIndex) {
            1 -> mCatImageView!!.setImageResource(R.drawable.chicken)
            2 -> mCatImageView!!.setImageResource(R.drawable.dog)
            3 -> mCatImageView!!.setImageResource(R.drawable.duck)

            else -> {
            }
        }
    }
}